### ISSUE TITLE
Helm prepare_helm_release.sh script execution error 'gzip: stdin: not in gzip format'

### ISSUE THREAD

**Issue by xcuseme123** [apache/pulsar-helm-chart#422]
**Describe the bug**
As per Apache Pulsar documentation [https://pulsar.apache.org/docs/3.1.x/getting-started-helm/#step-1-install-pulsar-helm-chart](url) , executed step3. Noticed an error in script 'gzip: stdin: not in gzip format'

**To Reproduce**
1. Install Git and GZip . Open GitBash
2. install minikube, virtualbox. Run below command
`   minikube start --memory=8192 --cpus=4 --kubernetes-version=1.28.3 --driver=virtualbox --no-vtx-check`
4. Run the script prepare_helm_release.sh
  ```
./scripts/pulsar/prepare_helm_release.sh \
      -n pulsar \
      -k pulsar-mini \
      -c
```
5. See error

**Expected behavior**
Script execution should complete without error

**Screenshots of error**

**Desktop (please complete the following information):**
 - OS: Windows 11

**Comment by xcuseme123** [comment]
![Uploading Helm script error.JPG…]()
Attaching error screen shot

**Comment by xcuseme123** [comment]
Error message:
generate the token keys for the pulsar cluster
Get pulsarctl install.sh script ...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  3987  100  3987    0     0   3613      0  0:00:01  0:00:01 --:--:--  3631
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     9    0     9    0     0     10      0 --:--:-- --:--:-- --:--:--    10

**gzip: stdin: not in gzip format**
tar: Child returned status 1
tar: Error is not recoverable: exiting now
generate the tokens for the super-users: proxy-admin,broker-admin,admin
generate the token for proxy-admin

**Comment by lhotari** [comment]
fixed by #191
